test/topology/conftest.py

```diff
@@ -216,7 +216,7 @@ def fails_without_consistent_cluster_management(request, check_pre_consistent_cl
 # unless the cluster is dirty or the test has failed.
 @pytest.fixture(scope="function")
 async def random_tables(request, manager):
-    tables = RandomTables(request.node.name, manager, unique_name())
+    tables = RandomTables(request.node.name, manager, unique_name(), 1)
```

Comment:
`test_custom` needs to be fixed in a similar way in this `test/pylib: configurable replication factor` commit.